ckanext/qdes/logic/action/get.py

```diff
@@ -264,7 +264,7 @@ def qdes_datasets_not_reviewed(context, config):
                 .get_point_of_contact(context, contact_point_pos) if contact_point_pos else {}
 
         rows.append({
-            'Dataset name': pkg_dict.get('title', pkg_dict.get('name', '')),
+            'Dataset ID': pkg_dict.get('id', pkg_dict.get('id', '')),
```

Comment:
We don't need two instances of pkg_dict.get('id') to set a default value of ''. Can be updated to `'Dataset ID': pkg_dict.get('id', '')`